app/models/restaurant.rb

```diff
@@ -1,4 +1,19 @@
 class Restaurant < ApplicationRecord
```

Comment:
We are using a gem called annotate to add the db scheme table structure as a comment above class. No need to add it here, just so you'll be familiar with it